src/styles/sidebar/components/selection-tabs.scss

```diff
@@ -48,3 +48,17 @@
 .selection-tabs__type--orphan {
   margin-left: -5px;
 }
+
+.selection-tabs__message {
+  border: 1px solid var.$grey-3;
+  padding: 3em;
+  text-align: center;
+}
+
+.selection-tabs__icon {
```

Comment:
Note: These styles are identical to the inline-icon styling for `VersionInfo`. Should pull them into a reusable utility class in a follow-up.